codebuild/bin/criterion_delta.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+set -eu
+source ./codebuild/bin/utils.sh
+
+make install
```

Comment:
Just a thought, weren't we trying to steer away from using make and instead start using cmake?

Reply:
Moving the integration tests to cmake is a larger effort/future pr.